lib/ansible/modules/web_infrastructure/django_manage.py

```diff
@@ -278,7 +307,12 @@ def main():
         if module.params[param]:
             cmd = '%s %s' % (cmd, module.params[param])
 
-    rc, out, err = module.run_command(cmd, cwd=app_path)
+    command_kwargs = {'cwd': app_path}
+    if 'output' in module.params and command in can_use_output_param_command:
+        command_kwargs['use_unsafe_shell'] = True
+        cmd = "%s > %s" % (cmd, pipes.quote(module.params['output']))
```

Comment:
* From the comment on line 305, it looks like this should be added to the parameters before end_of_command_params are added.
* I'd make use of django_manage's --output command: https://docs.djangoproject.com/en/2.0/ref/django-admin/#dumpdata  instead of resorting to use_unsafe_shell
* Doing that, though, means that we need to account for potential spaces in filenames without using pipes.quote.  We can do that by passing a list to module.run_command() instead of passing a string.

Altogether, those changes would look something like this:

``` python
cmd = ['./manage.py', command]

if command in noinput_commands:
    cmd.append('--noinput')

for param in general_params:
    if module.params[param]:
        cmd.append('--%s=%s' % (param, module.params[param]))

for param in specific_boolean_params:
    if module.boolean(module.params[param]):
        cmd.append('--%s' % param)

if 'output' in module.params and command in can_use_output_param_command:
    cmd = "%s --output=%s" % (cmd, module.params['output'])
rc, out, err = module.run_command(cmd, cwd=app_path)

# these params always get tacked on the end of the command
for param in end_of_command_params:
    if module.params[param]:
        cmd.append(module.params[param])
```

Reply:
@abadger Thank for review my code. 👍 
Accoring to #30045, cannot use `--output` option django version <1.8.
So, I use `stdout > filename` with `use_unsafe_shell=True` option instead of `--output`.

Reply:
I took a look at https://www.djangoproject.com/download/#supported-versions and it seems that django-1.8 is no longer a supported version.  I'd be okay with saying that this functionality is only available with django 1.8 or higher.  It looks like you can run
> ./manage.py --version

in order to capture the version of django that's being used.

I'd also be okay with simple documenting that we require django-1.8 and above since Django upstream no longer supports older versions, however, I don't know if  @scottanderson42 and @tastychutney would like to maintain support for older versions?